### PR TITLE
LEXUS NX 300 2020 Car Port

### PR DESCRIPTION
**Pull request by Goesreallyfast**
**Checklist**
- [x] added to README
- [x] test route added to [test_routes.py](../../selfdrive/test/test_routes.py)
- [x] route with openpilot:3fd5305f8b6ca765|2021-04-28--19-26-49
- [x] route with stock system:3fd5305f8b6ca765|2021-04-28--12-23-55

### CODE REVIEW

**Comment by Goesreallyfast** [comment]
++Co Contributers nelsonjchen VirtuallyChris vanillagorillaa

File: selfdrive/car/toyota/values.py [hunk 1]
```diff
@@ -1370,6 +1374,23 @@ class CAR:
     (Ecu.fwdCamera, 0x750, 0x6d): [
       b'8646F7801100\x00\x00\x00\x00',
     ],
+  },
+    CAR.LEXUS_NX_TSS2: {
```
Comment (adeebshihadeh) [inline selfdrive/car/toyota/values.py:1378]: fix the indentation

File: selfdrive/test/test_routes.py [hunk 2]
```diff
@@ -372,6 +372,11 @@
     'enableCamera': True,
     'enableDsu': False,
   },
+  "3fd5305f8b6ca765|2021-04-28--12-28-19": {
```
Comment (adeebshihadeh) [inline selfdrive/test/test_routes.py:375]: this route isn't fully uploaded. you'll need to connect your device to wifi for it to upload

File: selfdrive/car/toyota/values.py (hunk 1 above)
Comment (Goesreallyfast) [reply selfdrive/car/toyota/values.py]: The indentation should be corrected now, it had an extra space.

File: selfdrive/test/test_routes.py (hunk 2 above)
Comment (Goesreallyfast) [reply selfdrive/test/test_routes.py]: I reran the test route and updated the run name.

File: selfdrive/test/test_routes.py [hunk 3]
```diff
@@ -372,7 +372,7 @@
     'enableCamera': True,
     'enableDsu': False,
   },
-  "3fd5305f8b6ca765|2021-04-28--12-28-19": {
+  "3fd5305f8b6ca765|2021-04-28--19-26-49": {
```
Comment (adeebshihadeh) [inline selfdrive/test/test_routes.py:375]: this one is also not uploaded

Comment (Goesreallyfast) [reply selfdrive/test/test_routes.py]: Will this one work?
3fd5305f8b6ca765|2021-04-28--18-50-15

Comment (Goesreallyfast) [reply selfdrive/test/test_routes.py]: I redid the test route, it was after a longer drive. I turned off the car before I started the test and on the useradmin it shows the 2 separate drives, on Cabana it combined them into one

Comment (adeebshihadeh) [reply selfdrive/test/test_routes.py]: None of your routes are fully uploaded. make sure you have the upload raw toggle enabled in the settings.

Comment (Goesreallyfast) [reply selfdrive/test/test_routes.py:375]: Ah, I didn't even think about that! Apologies, will it upload the previous drives after I enable the toggle, or do I need to run the test route again with the setting enabled?

Comment (Goesreallyfast) [reply selfdrive/test/test_routes.py:375]: Looks like the drive should be uploaded now!

**Pull request merged**